deltacat/compute/converter/utils/converter_session_utils.py

```diff
@@ -56,7 +60,52 @@ def construct_iceberg_table_prefix(
     return f"{iceberg_warehouse_bucket_name}/{iceberg_namespace}/{table_name}/data"
 
 
-def partition_value_record_to_partition_value_string(partition):
-    # Get string representation of partition value out of Record[partition_value]
-    partition_value_str = partition.__repr__().split("[", 1)[1].split("]")[0]
-    return partition_value_str
+def group_all_files_to_each_bucket(
+    data_file_dict, equality_delete_dict, pos_delete_dict
+):
+    convert_input_files_for_all_buckets = []
+    files_for_each_bucket_for_deletes = defaultdict(tuple)
+    if equality_delete_dict:
+        # files_for_each_bucket contains the following files list:
+        # {partition_value: [(equality_delete_files_tuple_list, data_files_tuple_list, pos_delete_files_tuple_list)]
+        for k, v in data_file_dict.items():
```

Comment:
I think we also have some debug logs used for testing to clean up / remove here?